src/MethodsTest.java

```diff
@@ -0,0 +1,95 @@
+import java.util.Scanner;
+
+public class MethodsTest {
+    public static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        vowelConsonant();
+        System.out.println("\n");
+        priceAndDiscount();
+        System.out.println("\n");
+        powerOfNumber(3, 4);
+        System.out.println("\n");
+        palindrome("radar");
+        System.out.println("\n");
+        sumOfDigits();
+        System.out.println("\n");
+        numberOfVowels("sanja");
+        System.out.println("\n");
+        reverseNumber();
+
+    }
+
+    public static void vowelConsonant() {
+        System.out.println("Enter a character: ");
+        char ch = scanner.next().charAt(0);
+        if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u') {
+            System.out.println("Character is an vowel");
+        } else {
+            System.out.println("Character is a consonant");
+        }
+    }
+
+    public static void priceAndDiscount() {
+        int price, discount, finalPrice;
+        System.out.println("Enter price of product: ");
+        price = scanner.nextInt();
+        System.out.println("Enter discount of product: ");
+        discount = scanner.nextInt();
+        finalPrice = (price * discount) / 100;
```

Comment:
final price is price - (price * discount / 100).
This way you wrote it it returns 15 for 100e price and 15% discount but should return 85 instead